src/pg_cron.c

```diff
@@ -1888,7 +1893,10 @@ GetBgwTaskFeedback(shm_mq_handle *responseq, CronTask *task)
 						if (edata.elevel >= ERROR)
 							UpdateJobRunDetail(task->runId, NULL, GetCronStatus(CRON_STATUS_FAILED), display_msg.data, NULL, &end_time);
 						else
-							UpdateJobRunDetail(task->runId, NULL, GetCronStatus(CRON_STATUS_SUCCEEDED), display_msg.data, NULL, &end_time);
+							UpdateJobRunDetail(task->runId, NULL,
+											   running ? NULL : GetCronStatus(CRON_STATUS_SUCCEEDED),
+											   display_msg.data, NULL,
+											   running ? NULL : &end_time);
```

Comment:
would be slightly nicer to separate this out as `else if (running) ... else ...`

Reply:
agree, PR updated that way. Thanks!